.github/workflows/tests-template.yaml

```diff
@@ -6,16 +6,17 @@ on:
       arch:
         required: true
         type: string
-      runs-on:
-        required: true
-        type: string
+      # runs-on:
+      #   required: true
+      #   type: string
 permissions: read-all
 
 jobs:
   test:
-    runs-on: ${{ inputs.runs-on }}
     # this is to prevent arm64 jobs from running at forked projects
-    if: inputs.arch == 'amd64' || github.repository == 'etcd-io/etcd'
+    if: inputs.arch == 'arm64' || github.repository == 'etcd-io/etcd'
+    # this is to tune memory allocation of arm64 architecture
+    runs-on: ${{ if inputs.arch == 'arm64' then 'actuated-arm64-8cpu-8gb' else inputs.runs-on end }}
```

Comment:
I don't believe this file needs to be touched by this pr.

My understanding is the files we need to update (in addition to the `e2e-arm64.yaml` completed above) are:

https://github.com/etcd-io/etcd/blob/a49a867960034f61da4165f6873b2557392a2939/.github/workflows/tests.yaml#L15

https://github.com/etcd-io/etcd/blob/a49a867960034f61da4165f6873b2557392a2939/.github/workflows/robustness.yaml#L21

https://github.com/etcd-io/etcd/blob/a49a867960034f61da4165f6873b2557392a2939/.github/workflows/robustness-nightly.yaml#L30

https://github.com/etcd-io/etcd/blob/a49a867960034f61da4165f6873b2557392a2939/.github/workflows/robustness-nightly.yaml#L46

The update should be simply replacing `32gb` with `8gb`.